src/ApiGenerator/RestSpecification/Core/cat.allocation.json

```diff
@@ -61,6 +61,10 @@
         "type":"time",
         "description":"Explicit operation timeout for connection to master node"
       },
+      "cluster_manager_timeout":{
```

Comment:
do we want to remove "master_timeout" above?  I assumed "rename" means that you want to remove the old one...

Reply:
No, it is kept for compatibility with OpenSearch 1.x.
Actually 2.0 still allows to use `master_timeout`, but it is in deprecated state.